WordPress/Classes/ViewRelated/Post/PostEditorNavigationBarManager.swift

```diff
@@ -37,7 +37,8 @@ class PostEditorNavigationBarManager {
         let button = UIButton(type: .system)
         button.setImage(image, for: .normal)
         button.frame = CGRect(origin: .zero, size: image.size)
-        button.accessibilityLabel = NSLocalizedString("More", comment: "Action button to display more available options")
+        button.accessibilityLabel = NSLocalizedString("More Post Options", comment: "Action button to display more available options")
```

Comment:
We'll probably need a label that doesn't specifically mention "Post" here, since this button is also used in the page editor.